### PR TITLE
fifth part of the story

### PR DESCRIPTION
**Pull request by shree404**
7th part of the story

### CODE REVIEW

**Comment by shree404** [comment]
8th part of the story

**Comment by thenishantsapkota** [comment]
@shree404 Can you resolve the merge conflicts?
Thank you :+1:

File: story.md [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
+
+As he was typing away, he suddenly realized that he had been staring at the same line of code for several minutes without making any progress. He rubbed his eyes and looked closer at the screen, but the code still didn't make any sense to him.
+
+Feeling frustrated, Bob decided to take a break and get some fresh air. As he was walking outside, he saw a group of kids playing with a toy robot.
+
+Suddenly, an idea struck him. He could program the robot to help him with his project! He excitedly ran over to the kids and asked if he could borrow their robot.
+
+Bob quickly got to work and wrote a program that would allow the robot to type out code for him. He sat back and watched as the robot's tiny arms moved across the keyboard, typing out lines of code faster than he ever could.
+
+Feeling embarrassed, Bob had to spend the next several hours fixing all of the mistakes that the robot had made. From that day on, he never let his colleagues forget that even the most perfect programmer could still make mistakes - especially when they involved toy robots.
```
Comment (thenishantsapkota) [inline story.md:20]: Part of the story is missing here.
Can you fix that?

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

**Review by thenishantsapkota — APPROVED** [review]

**Comment by thenishantsapkota** [comment]
@12anupa-baral @jsurya860 Can you guys review this PR?
Thank you!

File: story.md [hunk 2]
```diff
@@ -1,15 +1,23 @@
...
+
+Bob quickly got to work and wrote a program that would allow the robot to type out code for him. He sat back and watched as the robot's tiny arms moved across the keyboard, typing out lines of code faster than he ever could.
+
+But as he was admiring the robot's work, he suddenly realized that the code it was typing was complete gibberish! He had forgotten to check the code before letting the robot loose.
+
+Bob quickly tried to shut down the program, but it was too late. The robot had already started wreaking havoc on his project, causing error messages to pop up all over the screen.
+
+Feeling embarrassed, Bob had to spend the next several hours fixing all of the mistakes that the robot had made. From that day on, he never let his colleagues forget that even the most perfect programmer could still make mistakes - especially when they involved toy robots.
+
+The incident with the robot taught Bob a valuable lesson about the importance of taking breaks and double-checking his work. He realized that sometimes, even the best programmers need to step away from their work to avoid burnout and ensure that their code is accurate.
```
Comment (jsurya860) [inline story.md:22]: Is this the conclusion or the missing part of the story?

**Review by jsurya860 — APPROVED** [review]
all great but little confused in the last part

**Pull request merged**